src/main/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePlugin.java

```diff
@@ -86,18 +77,21 @@ public boolean hasFailed(String cloudState) {
 
 	@Override
 	public String requestInstance(ComputeOrder computeOrder, CloudUser cloudUser) throws FogbowException {
```

Comment:
you could extract the request creation to an aux method in order to decrease the main method's size. Also, we're trying to follow a pattern where we create another aux method usually named as do**Request. This method wraps some operations responsible for really request the instance, in this case, and we just return its call...

Reply:
thanks for the heads up. will do